qiskit_ibm_runtime/base_primitive.py

```diff
@@ -132,17 +132,18 @@ def _run(self, pubs: Union[list[EstimatorPub], list[SamplerPub]]) -> RuntimeJob:
         primitive_inputs = {"pubs": pubs}
         options_dict = asdict(self.options)
         self._validate_options(options_dict)
-        primitive_inputs.update(self._options_class._get_program_inputs(options_dict))
+        primitive_options = self._options_class._get_program_inputs(options_dict)
+        primitive_inputs.update(primitive_options)
```

Comment:
I'm not sure what you meant. This will output something like

```
{"pubs": ..., "options": {...}, "resilience_level": 1}
```

to match what you have in your PR.

Reply:
Oh great, I just misunderstood your code.